notebooks/02-listas_y_for_loops.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Listas y for loops\n",
```

Comment:
<p>Acá! jeje</p><p> </p><p>Me gustó mucho que uses una lista de variables en vez de los meses. Habría que cambiarlos en el enunciado también.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoLatinas/Intro-to-python/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Listo! Thank you for spotting that.